src/NHSD.BuyingCatalogue.Database/NHSD.BuyingCatalogue.Database.sqlproj

```diff
@@ -163,11 +163,18 @@
     <None Include="PostDeployment\InsertPublicationStatuses.sql" />
     <None Include="PostDeployment\InsertSolutionCapabilityStatuses.sql" />
     <None Include="PostDeployment\InsertSolutionEpicStatuses.sql" />
+    <None Include="PostDeployment\ProdLikeData\MergeSuppliers.sql" />
+    <None Include="PostDeployment\ProdLikeData\MergeCatalogueItems.sql" />
+    <None Include="PostDeployment\ProdLikeData\MergeSolutions.sql" />
+    <None Include="PostDeployment\ProdLikeData\MergeAdditionalServices.sql" />
+    <None Include="PostDeployment\ProdLikeData\MergeAssociatedServices.sql" />
+    <None Include="PostDeployment\ProdLikeData\MergeMarketingContacts.sql" />
+    <None Include="PostDeployment\ProdLikeData\MergeSolutionEpics.sql" />
+    <None Include="PostDeployment\ProdLikeData\MergeSolutionCapabilities.sql" />
+    <None Include="PostDeployment\ProdLikeData\MergeFrameworkSolutions.sql" />
+    <None Include="PostDeployment\ProdLikeData\MergeCataloguePrices.sql" />
   </ItemGroup>
   <ItemGroup>
     <PostDeploy Include="PostDeployment\PostDeployment.sql" />
   </ItemGroup>
-  <ItemGroup>
-    <PreDeploy Include="PreDeployment\PreDeployment.sql" />
-  </ItemGroup>
 </Project>
```

Comment:
VS always removes it after a change but ideally this should end with newline to keep git happy – I'll leave it up to you to decide whether to make the change.

Reply:
Going to leave it.